### PR TITLE
Add blog

### PR DESCRIPTION
**Pull request by caicancai**

### CODE REVIEW

**Comment by caicancai** [comment]
The article has been published on the official Streampark public account

**Comment by caicancai** [comment]
cc @wolfboys

File: blog/StreamPark 在顺网科技的大规模生产实践.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+![图片](https://mmbiz.qpic.cn/mmbiz_png/8LQOHpLWEkEhXheAYFK1sfJjCjgbtI4NxHegtXL80u2K6Nzia5hszBRfNpYkfEn5geibnEjwX2J4Giah4wYPzI0AQ/640?wx_fmt=png&tp=wxpic&wxfrom=5&wx_lazy=1&wx_co=1)
+
+但是后续开发同学在使用过程中，发现 Streaming-Launcher 存在比较多的缺陷：Flink 开发成本依然过高、工作效率低下、问题排查困难。我们总结了 Streaming-Launcher 存在的缺陷，大致如下：
+
+### **SQL开发流程繁琐**
+
+作业务开发需要多个工具完成一个 SQL 作业开发，提高了开发同学的使用门槛。
+
+![cc0b1414ed43942e0ef5e9129c2bf817](D:\微信\WeChat Files\wxid_21agvh8tmps422\FileStorage\Temp\cc0b1414ed43942e0ef5e9129c2bf817.png)
```
Comment (wolfboys) [inline blog/StreamPark 在顺网科技的大规模生产实践.md:51]: image url is incorrect

**Comment by wolfboys** [comment]
Thanks for your hard work. There is a minor issue. All the image url are coming from an external link. We need to have the images hosted on our website and use our own links. 💪

**Comment by caicancai** [comment]
> Thanks for your hard work. There is a minor issue. All the image url are coming from an external link. We need to have the images hosted on our website and use our own links. 💪


Thank you for your review, I will modify this question later

**Comment by caicancai** [comment]
cc @wolfboys ，Tested

**Pull request merged**